process/working_groups.md

```diff
@@ -6,24 +6,27 @@ parent: Processes
+
+Working groups should operate within the scope of the mission, goals, non-goals, and deliverables listed in the approved charter. Any changes in scope must be reflected in the working group charter and approved by the TAC.
 
-If you would like to submit a working group proposal, please submit the [template] to the TAC. Working groups can begin formation to complete the [template] before TAC approval, but will not have official designation as an ASWF Working Group until TAC approval. You will also have to present to the ASWF TAC and wider community before your WG proposal will be voted upon by the TAC. 
+## Review
 
-## FAQs
+Working Groups will present to the TAC it's progress towards the mission, goals, non-goals, and deliverables listed in the approved charter after 6 months from when the working group was formally approved. This is an opportunity for the working group to elevate any concerns or needs from the TAC to complete it's mission, goals, non-goals, and deliverables. If there is insufficent progress within the working group, the TAC may opt to close the working group with a 2/3 majority vote of [TAC member]s.
 
-### Q: Who can propose a new working group?
-A: Anyone can propose a new working group ( whether from an ASWF member organization or not ), but it must be sponsored by a [TAC member].
+After one year, the working group is expected to have completed the mission, goals, non-goals, and deliverables listed in the approved charter. If the working group has not completed it's mission, goals, non-goals, and deliverables, and it's of the opinion of the TAC that it can be completed, the TAC may opt to extend the working group for up to 1 year with a 2/3 majority vote of [TAC member]s. 
 
-### Q: Can working groups have infrastructure during formation to enable collaboration?
-A: Yes! Please submit a [request on the foundation repo](https://github.com/AcademySoftwareFoundation/foundation/issues/new) and we can provide a mailing list, Slack channel, and GitHub repo.
+## Long-term working group
 
-### Q: What if the goals or mission of the working group change?
-A: The TAC recognizes that in the lifecycle of a working group goals or mission may change. In those cases, the working group must present to the TAC an updated version of their [template] for approval.
+The TAC may also opt to establish the working group as a long-term working group after it's 1 year anniversary with a 2/3 majority vote of [TAC member]s, if it is deemed that such a working group is essential for the sustainability of the Academy Software Foundation. It is expected that a long-term working group has it's charter reviewed and revisted annually with the approval of the TAC.
```

Comment:
it's -> its (2x)

This addresses the concerns about "long term working groups" well.

Reply:
Fixed typo - thanks for the feedback on that statement!